ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -907,53 +903,6 @@ func (r *RayServiceReconciler) updateRayClusterInfo(rayServiceInstance *rayv1.Ra
 	}
 }
 
-// TODO: When start Ingress in RayService, we can disable the Ingress from RayCluster.
-func (r *RayServiceReconciler) reconcileIngress(ctx context.Context, rayServiceInstance *rayv1.RayService, rayClusterInstance *rayv1.RayCluster) error {
-	if rayClusterInstance.Spec.HeadGroupSpec.EnableIngress == nil || !*rayClusterInstance.Spec.HeadGroupSpec.EnableIngress {
```

Comment:
Also deprecate the `.Spec.HeadGroupSpec.EnableIngress` field?

Reply:
This is also used by the RayCluster controller. I will deprecate the field if we decide to deprecate the built-in Ingress support of RayCluster. 

https://github.com/ray-project/kuberay/blob/0575bd16ceb61f4c148ada7615f28a7c0f45b105/ray-operator/controllers/ray/raycluster_controller.go#L414